rpi: update config.txt

jdorigao opened the pull request.

dmanlfc commented:
This will break RPi bespoje cases & GPIO controllers on each upgrade as you're not allowing previous settings to remain after an upgrade...

jdorigao commented:
The ideal would be to save these settings in the userconfig.txt file or we can add a file just for the extra settings added by Batocera such as gpio.
The config.txt file should be changed by batocera, because any change to it depends on the end user doing a clean install or changing the file.

dmanlfc commented:
Ok can you adjust the configurations that touch the config.txt. I'm on holiday, so no access to a dev PC. Obviously you will need to add the userconfig.txt to the array.

jdorigao commented:
@dmanlfc @nadenislamarre I changed to make this change in two steps in this first step I updated the config.txt file separating the options that can be changed by the user and the files are not being overwritten.
The second step will be done in future versions of Batocera, which is to overwrite the config.txt files.
